subsys/bluetooth/controller/ll_sw/ll_addr.c

```diff
@@ -55,3 +56,8 @@ u32_t ll_addr_set(u8_t addr_type, u8_t const *const bdaddr)
 
 	return 0;
 }
+
+void bt_ctlr_set_public_addr(const u8_t *addr)
+{
+    (void)memcpy(pub_addr, addr, sizeof(pub_addr));
```

Comment:
we dont prefix (void) anymore in Bluetooth subsys, do we?

Reply:
I thought that's the convention for the whole tree in spirit of of attaining MISRA compliance some day. If you do a grep for memset you'll see that essentially the entire tree has been converted to do this (only exception is ext/)

Reply:
ok